docs/requirements.txt

```diff
@@ -1,2 +1,2 @@
-sphinx
+sphinx<1.6.1
```

Comment:
Can a comment be added with a URL to your Sphinx PR? That way we know when to remove the workaround.